tests/BlazingSQLTest/EndToEndTests/TestSuites/nullaryOpsSuite.yaml

```diff
@@ -0,0 +1,25 @@
+# Nullary ops Suite
+
+TEST_SUITE:
+  SETUP:
+    EXPECTED_SOURCE: python_script
+    APPLY_ORDER: True
+    USE_PERCENTAGE: False
+    ACCEPTABLE_DIFFERENCE: 0.01
+    COMPARE_WITH: drill
+    SKIP_WITH:
+      - JSON
+      - CSV
+      - CUDF
+
+  TEST_01: # current_date is not implemented as a nullaryOps, but need to be considered
```

Comment:
Is current_date and current_timestamp something that is not supported by drill or pyspark? why do we have to validate it this way?

Reply:
The e2e framework has the _mode_ `full` to perform the queries using drill or pyspark. But, that mode cannot be executed in gpuci. So we need a way to get result-queries for cases like `select current_date from ...` when the result need to be computed.
There is a task to implement that. This changes are a first approach.

Reply:
For `select current_date` i can see how you cant do that in the CI framework using previously calculated results, but that is not what you have here. Here you can use previously calculated results

Reply:
Indeed, it is. Than those queries because I found an error in the e2e: the result of the query changes if ORC or PARQUET is used for the creation of the table.

However, I have added the original queries again, since I was able to reproduce the error using the following script.

```python
from blazingsql import BlazingContext
import cudf

bc = BlazingContext()

bc.create_table('orders_parquet',
    ['.../blazingsql-testing-files/data/tpch/orders_0_0.parquet',
     '.../blazingsql-testing-files/data/tpch/orders_1_0.parquet']
)

bc.create_table('orders_orc', 
    ['.../blazingsql-testing-files/data/tpch/orders_0_0.orc',
     '.../blazingsql-testing-files/data/tpch/orders_1_0.orc']
)

print('>>> creado')

def run(tableName):
  query = f'''select o_orderdate
    from {tableName}
    where cast(o_orderdate as date) < CURRENT_DATE order by o_orderdate limit 5'''

  print('>>> explain')
  print(bc.explain(query))

  print('>>> sql query')
  df = bc.sql(query)

  print('>>> result')
  print(df)
  print(df.dtypes)

print('++++++++++++++++++++++++++++++++++++++++')
print('CON PARQUET')
print('++++++++++++++++++++++++++++++++++++++++')
run('orders_parquet')
print('\n++++++++++++++++++++++++++++++++++++++++')
print('CON ORC')
print('++++++++++++++++++++++++++++++++++++++++')
run('orders_orc')```